app/assets/javascripts/controllers/toolbar_controller.js

```diff
@@ -14,6 +14,22 @@
   */
   function subscribeToSubject() {
     ManageIQ.angular.rxSubject.subscribe(function(event) {
+      if (event.eventType === 'sendCountSelectedAndUpdateToolbar') {
+        this.MiQToolbarSettingsService.countSelected = event.countSelected;
```

Comment:
..and I wonder if the name could be a bit simpler .. like `updateToolbarCount` or something.. WDYT?